src/pages/products/sgid/environment/petroleum-storage-tanks.astro

```diff
@@ -0,0 +1,115 @@
+---
+import BulletedList from '@components/page/BulletedList.astro';
+import ExternalLink from '@components/page/ExternalLink.astro';
+import Layout from '@layouts/DataPage.astro';
+
+import type { IMetadata, IPageMetadata } from '@models/products/sgid/types';
+import { ProductType, SgidCategory } from '@models/products/sgid/types';
+
+import HubDownloads from '@components/data/HubDownloads.astro';
+import Metadata from '@components/data/Metadata.astro';
+import UpdateHistory from '@components/data/UpdateHistory.astro';
+
+import CardWithPopularLink from '@components/page/CardWithPopularLink.astro';
+import CardWithSmallLink from '@components/page/CardWithSmallLink.astro';
+import GridForMoreResources from '@components/page/GridForMoreResources.astro';
+import GridForYouMightLike from '@components/page/GridForYouMightLike.astro';
+import Section from '@components/page/Section.astro';
+
+import Contacts from '@components/page/Contacts.astro';
+import { dataPages } from '@data/downloadMetadata';
+
+export const metadata: IMetadata = {
+  title: 'Utah Petroleum Storage Tank Facilities',
+  description:
+    'Utah facility information for each petroleum storage tank (PST). This is the facility where the storage tanks are located.',
+  stewards: ['DEQ DERR'],
+  type: ProductType.POINT,
+  category: SgidCategory.ENVIRONMENT,
+};
+
+const page: IPageMetadata = {
+  ...metadata,
+  application: 'https://enviro.deq.utah.gov',
+  updateHistory: ['Nightly'],
+  hub: {
+    ...dataPages[metadata.title],
+  },
+};
+---
+
+<Layout title={page.title} subTitle={page.updateHistory[0]} category={page.category} actionUrl={page.application}>
+  <Metadata slot="metadata" {...page} />
+
+  <Fragment slot="summary">
+    <p>
+      Utah facility information for each petroleum storage tank (PST). This is the facility where the storage tanks are
+      located.
+    </p>
+
+    <p>Additional tank information can be found in these related tables:</p>
+    <BulletedList>
+      <li>
+        <ExternalLink href="https://opendata.gis.utah.gov/datasets/utah::utah-petroleum-storage-tanks-compliance/about"
+          >Tanks Compliance</ExternalLink
+        >
+      </li>
+      <li>
+        <ExternalLink href="https://opendata.gis.utah.gov/datasets/utah::utah-petroleum-storage-tanks/about"
+          >Tanks</ExternalLink
+        >
+      </li>
+      <li>
+        <ExternalLink href="https://opendata.gis.utah.gov/datasets/utah::utah-deqmap-lust/about">LUST</ExternalLink>
+      </li>
+    </BulletedList>
+  </Fragment>
+
+  <Section title="Use the data" slot="downloads" titlePosition="center">
+    <HubDownloads {...page.hub} />
+  </Section>
+
+  <Fragment slot="description">
+    <p>
+      This dataset is updated nightly via forklift from the DERR database which is continually updated. This data layer
```

Comment:
**question**: why is this the only dataset that gets the forklift mention?

Reply:
I'm not sure other than I was just copying from the old website and internal metadata. Do you think that all of the datasets that we update via forklift should have that in the notes?

Reply:
I am happy with consistency. So all or nothing makes sense to me. If you choose all we can make a partial to include. Otherwise it's an implementation detail we can hide. Your choice.